src/storage/filesystem.py

```diff
@@ -7,6 +7,7 @@
 from os import listdir, makedirs, path, remove, rmdir
 from threading import RLock
 
+from debug import logger
```

Comment:
bad practice, please read the `debug` doc

Reply:
I don't understand, what's wrong here? This is like it's used everywhere else in the project.

Reply:
Another import from outer package. When I refactored the logger I left the [note](https://pybitmessage.rtfd.io/en/v0.6/autodoc/pybitmessage.debug.html):
> The old form: from debug import logger is also may be used, but only in the top level modules.

Reply:
I see.

Reply:
I have replaced debug with logging import